gammapy/modeling/models/spectral.py

```diff
@@ -427,7 +428,9 @@ def plot(
         Parameters
         ----------
         energy_bounds : `~astropy.units.Quantity`
-            Plot energy bounds passed to MapAxis.from_energy_bounds.
+            Plot energy bounds passed to `~gammapy.maps.MapAxis.from_energy_bounds`.
```

Comment:
The description with  gammapy.maps.MapAxis.from_energy_bounds is very technical and only experts can understand...
Also you have not mentioned the default values